packages/flutter_tools/lib/src/flutter_cache.dart

```diff
@@ -158,16 +158,13 @@ class MaterialFonts extends CachedArtifact {
 ///
 /// This SDK references code within the regular Dart sdk to reduce download size.
 class FlutterWebSdk extends CachedArtifact {
-  FlutterWebSdk(Cache cache, {required Platform platform})
-   : _platform = platform,
-     super(
+  FlutterWebSdk(Cache cache, {@Deprecated('The platform argument is deprecated') Platform? platform})
```

Comment:
Can we not just remove this field?

Reply:
For more context, as flutter_tools is not a public API, we generally do not deprecate function parameters, but just remove them (if Google testing is failing, the process would be creating a g3fix for it).

Reply:
Thanks, I went back to just removing platform and will create the g3fix once the CL is available in the frob ui

Reply:
Thanks!